### ISSUE TITLE
DCGAN's Discriminator : lack of nn.BatchNorm2d(ndf) 

### ISSUE THREAD

**Issue by StevenJokes** [pytorch/tutorials#1126]
I'm learning DCGAN by translating [mxnet](http://preview.d2l.ai/d2l-en/master/chapter_generative-adversarial-networks/dcgan.html)  code to [pytorch](https://pytorch.org/tutorials/beginner/dcgan_faces_tutorial.html).

<img width="1272" alt="Snipaste_2020-08-16_20-15-13" src="https://user-images.githubusercontent.com/25657787/90334221-c8192600-dffe-11ea-95ab-9e42a1ddf8eb.png">
I'm a newbie.
Then in my guess,  Discriminator has pretty much structures opposite to Generator.
So they must have same number layers.

Here is my code:
https://github.com/StevenJokes/d2l-en-read/blob/master/chapter-generative-adversarial-networks/dcgan_pt3.ipynb
My PR: https://github.com/pytorch/tutorials/pull/1125
